bootstrap/frontend/src/main/scala/io/udash/bootstrap/form/UdashForm.scala

```diff
@@ -82,11 +84,12 @@ object UdashForm {
     * More: <a href="http://getbootstrap.com/css/#forms-inline">Bootstrap Docs</a>.
     *
     * @param componentId Id of the root DOM node.
+    * @param onSubmit Callback executed when a form is submitted
     * @param content Form content
     * @return `UdashForm` component, call render to create DOM element.
     */
-  def inline(componentId: ComponentId, content: Modifier*): UdashForm =
-    new UdashForm(Some(BootstrapStyles.Form.formInline), componentId)(content)
+  def inline(componentId: ComponentId, onSubmit: Event => Any, content: Modifier*): UdashForm =
```

Comment:
What do you think about moving `onSubmit` to a separate argument list? It feels more natural, a vararg can be confusing enough for a single list.